Userland/Libraries/LibJS/Runtime/ArrayPrototype.cpp

```diff
@@ -85,6 +86,7 @@ void ArrayPrototype::initialize(GlobalObject& global_object)
     // 23.1.3.35 Array.prototype [ @@unscopables ], https://tc39.es/ecma262/#sec-array.prototype-@@unscopables
     // With find from last proposal, https://tc39.es/proposal-array-find-from-last/#sec-array.prototype-@@unscopables
     // With array grouping proposal, https://tc39.es/proposal-array-grouping/#sec-array.prototype-@@unscopables
+    // With change array by copy proposal, https://tc39.es/proposal-change-array-by-copy/#sec-array.prototype.with
```

Comment:
All the other links here refer to the @@unscopables section so should this link be:
https://tc39.es/proposal-change-array-by-copy/#sec-array.prototype-@@unscopables
?